src/Shared/FileUtilities.cs

```diff
@@ -276,140 +347,40 @@ private unsafe static bool AreStringsEqual(char* buffer, int len, string s)
             return true;
         }
 
-        /// <summary>
-        /// Gets the canonicalized full path of the provided path.
-        /// Path.GetFullPath The pre .Net 4.6.2 implementation of Path.GetFullPath is slow and creates strings in its work.
-        /// Therefore MSBuild has its own implementation on full framework.
-        /// Guidance for use: call this on all paths accepted through public entry
-        /// points that need normalization. After that point, only verify the path
-        /// is rooted, using ErrorUtilities.VerifyThrowPathRooted.
-        /// ASSUMES INPUT IS ALREADY UNESCAPED.
-        /// </summary>
-        internal static string NormalizePath(string path)
+        private static bool IsUNCPath(string path)
```

Comment:
Does this need to do anything on non-Windows? Can we just short-circuit it there?

Reply:
Nope, so far it's only being used under the ```isWindows``` check above but that probably belongs here too.